tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -648,6 +648,39 @@ class FooClass
                 null,
                 ['ensure_fully_multiline' => true],
             ],
+            'test function argument with multiline echo in it' => [
+            <<<'UNAFFECTED'
+<?php
+call_user_func(function ($arguments) {
+    echo 'a',
+      'b';
+}, $argv);
+UNAFFECTED
+            ,
+                null,
+                ['ensure_fully_multiline' => true],
+            ],
+            'test function argument with oneline echo in it' => [
+            <<<'EXPECTED'
+<?php
+call_user_func(
+    function ($arguments) {
```

Comment:
I contributed nothing regarding indention in my feature. We established it was out of scope and that all was provided was a best effort to make the code PSR2 compliant, which I believe this is.